connaisseur/validators/notrayv1/notary.py

```diff
@@ -49,6 +50,28 @@ def __init__(self, name: str, host: str, pub_root_keys: list, **kwargs):
         self.pub_root_keys = pub_root_keys
         self.is_acr = kwargs.get("is_acr", False)
         self.is_cosign = kwargs.get("is_cosign", False)
+        self.auth = kwargs.get("auth", {})
```

Comment:
better specify the keyword arguments explicitly in the function signature, with default:

```python
def __init__(self, name: str, host: str, pub_root_keys: list, is_acr=False, is_cosign=False, auth={})
```